LICENSE.md

```diff
@@ -22,3 +22,8 @@ The GLM module is licensed under the MIT License:
 > LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
 > OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
 > WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+Note that the GLM module has an implicit dependency on the
```

Comment:
Not sure what "implicit" means here:
```suggestion
Note that the GLM module depends on the
```

Reply:
Hm, yeah, maybe there is a better word.  GLM.jl doesn't depend *directly* on Rmath, the dependency is via Distributions via StatsFuns.  Anyway, if you prefer I can take your suggestion, just let me know what you think now that I've clarified what I was going for. :)

Reply:
Let's say indirect then. :-)
```suggestion
Note that the GLM module has an indirect dependency on the
```